kedro-datasets/kedro_datasets_experimental/tests/langchain/test_chat_anthropic_dataset.py

```diff
@@ -0,0 +1,139 @@
+"""Tests for ChatAnthropic dataset classes."""
+
+from unittest.mock import Mock, patch
+
+import pytest
+from kedro.io import DatasetError
+
+from kedro_datasets_experimental.langchain._anthropic import ChatAnthropicDataset
+
+
+@pytest.fixture
+def anthropic_credentials():
+    """Fixture for standard Anthropic credentials."""
+    return {
+        "base_url": "https://api.anthropic.com/v1",
+        "api_key": "sk-ant-test-key"  # pragma: allowlist-secret
+    }
+
+
+@pytest.fixture
+def anthropic_kwargs():
+    """Fixture for ChatAnthropic kwargs."""
+    return {"model": "claude-instant-1", "temperature": 0.0}
+
+
+class TestAnthropicDataset:
+    """Test the ChatAnthropicDataset class."""
+
+    def test_init_with_credentials(self, anthropic_credentials, anthropic_kwargs):
+        """Test dataset initialization with credentials."""
+        dataset = ChatAnthropicDataset(credentials=anthropic_credentials, kwargs=anthropic_kwargs)
+
+        assert dataset.credentials == anthropic_credentials
+        assert dataset.kwargs == anthropic_kwargs
+
+    def test_init_without_credentials(self):
+        """Test dataset initialization without credentials (uses environment variables)."""
+        dataset = ChatAnthropicDataset()
+
+        assert dataset.credentials == {}
+        assert dataset.kwargs == {}
+
+    def test_init_with_partial_credentials(self):
+        """Test dataset initialization with partial credentials works."""
+        credentials = {"api_key": "sk-ant-test-key"}  # pragma: allowlist-secret
+        dataset = ChatAnthropicDataset(credentials=credentials)
+
+        assert dataset.credentials == credentials
+        assert dataset.kwargs == {}
+
+    def test_describe(self, anthropic_credentials, anthropic_kwargs):
+        """Test the _describe method returns kwargs."""
+        dataset = ChatAnthropicDataset(credentials=anthropic_credentials, kwargs=anthropic_kwargs)
+        description = dataset._describe()
+        assert description == {**{k: "***" for k in anthropic_credentials.keys()}, **anthropic_kwargs}
+
+    def test_save_raises_error(self, anthropic_credentials):
+        """Test that save method raises DatasetError."""
+        dataset = ChatAnthropicDataset(credentials=anthropic_credentials)
+        with pytest.raises(DatasetError, match="ChatAnthropicDataset is a read only dataset type"):
+            dataset.save(data="test")
+
+    @patch('kedro_datasets_experimental.langchain._anthropic.ChatAnthropic')
+    def test_load_with_credentials(self, mock_chat_anthropic, anthropic_credentials, anthropic_kwargs):
+        """Test that load method creates ChatAnthropic instance with credentials."""
+        mock_instance = Mock()
+        mock_chat_anthropic.return_value = mock_instance
```

Comment:
Minor: you could make a fixture of these mock instances similar to the OpikPromptDataset (https://github.com/kedro-org/kedro-plugins/blob/main/kedro-datasets/kedro_datasets_experimental/tests/opik/test_opik_prompt_dataset.py#L16)